Sources/BrowserServicesKit/Autofill/Matchers/AutofillAccountMatcher.swift

```diff
@@ -45,6 +46,7 @@ public struct AutofillWebsiteAccountMatcher: AutofillAccountMatcher {
         self.tld = tld
     }
 
+    @available(*, deprecated, message: "Use findMatches(accounts:for:) -- Which returns an already sorted list with removed duplicates")
```

Comment:
It will still be used by iOS, but I marked it as deprecated as at some point we'd likely want to migrate sorting and dupe removal to use the new, more complex methods implemented on macOS.